runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollPlugin.java

```diff
@@ -389,6 +392,18 @@ public void onGameTick(final GameTick event)
 		updateClue(findClueScroll());
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (event.getGroupId() < WidgetID.BEGINNER_CLUE_MAP_CHAMPIONS_GUILD
+			|| event.getGroupId() > WidgetID.BEGINNER_CLUE_MAP_WIZARDS_TOWER)
+		{
+			return;
+		}
+		// Beginner Map Clues all use the same ItemID, but the WidgetID used to display them is unique
```

Comment:
add empty line above, also, why is this comment here, it should be on forWidgetIt method or somewhere in the actual class